lib/sidekiq/processor.rb

```diff
@@ -234,5 +234,14 @@ def cloned(thing)
       Marshal.load(Marshal.dump(thing))
     end
 
+    def constantize(str)
+      names = str.split('::')
+      names.shift if names.empty? || names.first.empty?
```

Comment:
Or maybe the split behavior might have changed among Ruby versions. Is that possible. I didn't check in all Ruby supported versions...